scipy/optimize/optimize.py

```diff
@@ -629,6 +638,13 @@ def _minimize_neldermead(func, x0, args=(), callback=None,
             # only xtol was probably specified, use it.
             xatol = unknown_options['xtol']
         unknown_options.pop('xtol')
+    if 'initial_simplex' in unknown_options:
```

Comment:
Please remove this addition, I don't think we're going to be deprecating this keyword.